test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -731,21 +692,8 @@ public static void AssertUntrustedRoot(IEnumerable<ILogMessage> issues, LogLevel
 
         public static void AssertNotTimeValid(IEnumerable<ILogMessage> issues, LogLevel logLevel)
         {
-            string notTimeValid;
+            string notTimeValid = X509ChainStatusFlags.NotTimeValid.ToString();
```

Comment:
There is a flaw with this change.

While searching for the status flag name is more robust than searching for the status information, you need to ensure that you're searching for exactly what you expect.

For example, this error would incorrectly pass your validation:
```
NU0000: CtlNotTimeValid: Specifies that the certificate trust list (CTL) is not valid because of an invalid time value, such as one that indicates that the CTL has expired.
```

Reply:
Thanks for reviewing Damon :)
If I log it as 
`$"ChainStatus({chainStatus.Status}): {chainStatus.StatusInformation?.Trim()}"`
and verify $"ChainStatus({chainStatus.Status})"
Would that be strong enough?

Reply:
Two comments:
1.  This is a message that a user will see.  `ChainStatus` might be a name you use when programming, but it's not suitable for a user-friendly message.  Additionally, there should be a single space between a word and an open parenthesis symbol.

A better message would be something like:
```C#
$"The certificate chain status is {chainStatus.Status}:  {chainStatus.StatusInformation?.Trim()}"
```
Keep in mind that `X509ChainStatus.Status` can be a combination of flags.

2.  It is not a goal to write error messages to pass tests.  You can be a bit more clever in writing your test validation.  For example, try splitting the message on punctuation characters and then search for a match.
```C#
string notTimeValid = X509ChainStatus.NotTimeValid.ToString();
bool isNotTimeValid = message.Split(new[] { ' ', ':' }).Where(word => word == notTimeValid).Any();

Assert.True(isNotTimeValid);
```

Reply:
Thanks for your suggestions! Fixed.